subsys/zigbee/Kconfig

```diff
@@ -200,6 +200,24 @@ config ZIGBEE_UART_TX_BUF_LEN
 
 endif #ZIGBEE_HAVE_SERIAL
 
+menuconfig ZIGBEE_CLI_SHELL
+	bool "Enable Zigbee CLI Shell"
+	select SHELL
+
+if ZIGBEE_CLI_SHELL
+
+config ZIGBEE_CLI_SHELL_DEBUG_CMD
+	bool "Enable debug shell commands"
+
+config ZIGBEE_CLI_SHELL_PROMPT
+	string "Prompt to be used in Zigbee shells"
+
+config ZIGBEE_CLI_ENDPOINT
+	int "Number of endpoint to be used by the CLI instance"
```

Comment:
Do we need a default value here?

Reply:
We don't need default value here. The endpoint used by the CLI must be registered and it depends on application, so errors are logged if CLI endpoint is not registered or ZCL ctx is not registered.